### PR TITLE
RNA transcription

### PR DESCRIPTION
**Pull request by clementi**
Implement RNA transcription exercise.

### CODE REVIEW

File: rna-transcription/example.nim [hunk 1]
```diff
@@ -0,0 +1,19 @@
+proc toRna*(strand: string): string =
+  var rna = ""
+
+  for c in strand:
+    if c == 'A':
```
Comment (dom96) [inline rna-transcription/example.nim]: You should use a case statement here.

File: rna-transcription/example.nim [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      rna &= 'U'
+    elif c == 'G':
+      rna &= 'C'
+    elif c == 'C':
+      rna &= 'G'
+    elif c == 'T':
+      rna &= 'A'
+    else:
+      var err: ref ValueError
```
Comment (dom96) [inline rna-transcription/example.nim]: this can be shortened to: ``raise newException(ValueError, "Invalid nucleotide")``.

**Comment by kytrinyx** [comment]
This looks great, thank you! Would you mind squashing your commits?

**Comment by clementi** [comment]
@kytrinyx There, that ought to do it. Commits squashed.

**Comment by kytrinyx** [comment]
Fabulous, thank you! Merged in cadb21fd73896791edf2f42eac809c1f784953ba

**Pull request closed without merging**